ui/src/components/admin/Notifications/ProgressBanner.js

```diff
@@ -216,7 +216,7 @@ const ProgressBanner = ({ renderIcon }) => {
   return (
     <Notification type={getProgressBannerType()}>
       {renderIcon(getProgressBannerType())}
-      <Col>
+      <Col style={{ maxWidth: '50%', width: '100%' }}>
```

Comment:
Message details are quite lengthy for Publish Progress, so we add a max width to make the text wrap sooner and give the rest of the components (progress bar, stop button) room to breathe.